bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/classic/AdminClientClassicApi.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase.wrappers.classic;
+
+import com.google.api.core.ApiFuture;
+import com.google.api.core.InternalApi;
+import com.google.cloud.bigtable.admin.v2.models.CreateTableRequest;
+import com.google.cloud.bigtable.admin.v2.models.ModifyColumnFamiliesRequest;
+import com.google.cloud.bigtable.admin.v2.models.Table;
+import com.google.cloud.bigtable.core.IBigtableTableAdminClient;
+import com.google.cloud.bigtable.hbase.wrappers.AdminClientWrapper;
+import java.util.List;
+
+/** For internal use only - public for technical reasons. */
+@InternalApi("For internal usage only")
+public class AdminClientClassicApi implements AdminClientWrapper {
+
+  private final IBigtableTableAdminClient delegate;
```

Comment:
Please delegate to `BigtableTableAdminClient`. I really would like to avoid double wrapping and the I would like to delete the old wrappers

Reply:
Thanks for the suggestion!!
Updated this to delegate with core-client's `BigtableTableAdminClient`.